.astylerc

```diff
@@ -0,0 +1,57 @@
+### AStyle configuration options ###
+
+# Waterloo Rocketry does not enforce a team-wide style guide, but we do
+# ask that all codebases remain internally consistent. If you change
+# the options in this file, please rerun the formatter on all source
+# files using `./format.sh -a`
+
+# Documentation for AStyle and a description of what all options do
+# is at http://astyle.sourceforge.net/astyle.html.
+
+# Put opening braces on a new line after the expression, like this:
```

Comment:
K&R style doesn't put the opening braces on a new line after an expression, only after a function name. For example, astyle style=kr turns this:

```
int foo(void) { return 7; }

int main() { if (foo()) { return 1; } return 0; }
```

Into this:
```
int foo(void)
{
    return 7;
}

int main()
{
    if (foo()) {
        return 1;
    }
    return 0;
}
```

Reply:
Before anyone says that this is weird and wrong and inconsistent and therefore we shouldn't use k&r style, no. K&R style is still perfect in every way, and is still proper to use, because it's what K&R used.

Reply:
comment corrected

god I hate this style

@zx100x100 pls change it as soon as we graduate and can't yell at you about it anymore

Reply:
@zx100x100 hold strong, do what's right (k&r is the right style)

Reply:
\>:( I don't like k&r either but y'all do y'all

Reply:
which style has all of the {s in line with the functions and loops?
imo { doesn't deserve it's own line.